File: StoryFlow_Shared/Models/GetSentenceDto.cs
namespace StoryFlow_Shared.Models
{
    public class GetSentenceDto
    {
        public int Id { get; set; }
        public int StoryId { get; set; }
        public int? Order { get; set; }
        public string PolishMeaning { get; set; } = string.Empty;
        public string EnglishMeaning { get; set; } = string.Empty;
        public ICollection<GetUserSentenceDto> UserSentences { get; set; } = new List<GetUserSentenceDto>();
    }
}

File: StoryFlow_Client/src/app/models/get-sentence-dto.ts
import { GetUserSentenceDto } from "./get-user-sentence-dto";

export interface GetSentenceDto {
    id: number;
    storyId: number;
    order: number | null;
    polishMeaning: string;
    englishMeaning: string;
    userSentences: GetUserSentenceDto[];
}